action.yml

```diff
@@ -39,10 +39,18 @@ inputs:
     description: "Script to run on host machine before rsync"
     required: false
     default: ""
+  SCRIPT_BEFORE_REQUIRED:
+    description: "If true, the action will fail if the before script fails"
+    required: false
+    default: "false"
   SCRIPT_AFTER:
     description: "Script to run on host machine after rsync"
     required: false
     default: ""
+  SCRIPT_AFTER_REQUIRED:
+    description: "If true, the action will fail if the after script fails"
+    required: false
+    default: "false"
```

Comment:
Same as for SCRIPT_BEFORE_REQUIRED

Reply:
Thank you for your reminder. I will correct it.